test framework changes cause tests to fail (compileSDK28 / targetSDK28)

**Describe the bug:**
Tests fail when run using compileSDK28. See https://ci.cgeo.org/job/cgeo%20pull%20request/2307/console

eg:
```
14:07:29  > Task :main:compileBasicDebugAndroidTestJavaWithJavac FAILED
14:07:29  63 actionable tasks: 58 executed, 2 from cache, 3 up-to-date
14:07:29  /home/jenkins/slave/workspace/cgeo pull request/tests/src-android/cgeo/CGeoTestCase.java:9: error: package android.test does not exist
```

**Possible reason**
There are relevant changes to the Android testing framework with Android 9 / API 28, see https://developer.android.com/about/versions/pie/android-9.0-changes-all#android-test-changes

--- a/main/build.gradle
+++ b/main/build.gradle
@@ -323,9 +323,8 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp:3.12.5'
 
     // Play Services
-    def playServicesVersion = '11.8.0'
-    implementation "com.google.android.gms:play-services-location:$playServicesVersion"
-    implementation "com.google.android.gms:play-services-maps:$playServicesVersion"
+    implementation 'com.google.android.gms:play-services-location:16.0.0'
+    implementation 'com.google.android.gms:play-services-maps:16.1.0'
     // somehow there is a transitive play service dependency which we don't want
     configurations.all*.exclude module: "play-services-measurement"
 

--- a/tests/AndroidManifest.xml
+++ b/tests/AndroidManifest.xml
@@ -5,8 +5,8 @@
     android:versionName="1.0" >
 
     <uses-sdk
-        android:minSdkVersion="14"
-        android:targetSdkVersion="19" />
+        android:minSdkVersion="16"
+        android:targetSdkVersion="28" />
 
     <uses-feature
         android:name="android.hardware.screen.portrait"
@@ -19,8 +19,15 @@
     <application
         android:allowBackup="false"
         android:icon="@drawable/icon"
-        android:label="@string/app_name" >
-        <uses-library android:name="android.test.runner" />
+        android:label="@string/app_name">
+
+        <uses-library
+            android:name="android.test.base"
+            android:required="false" />
+
+        <uses-library
+            android:name="android.test.runner"
+            android:required="false" />
 
         <activity android:name="CgeoTestsActivity" >
             <intent-filter>

